src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { Route, Switch } from "react-router-dom";
+import { Route } from "react-router-dom";
 import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
these could be combined into a single line

Reply:
Done.